presto-hive/src/test/java/io/prestosql/plugin/hive/HiveQueryRunner.java

```diff
@@ -68,82 +70,94 @@ private HiveQueryRunner()
     private static final String TPCH_BUCKETED_SCHEMA = "tpch_bucketed";
     private static final DateTimeZone TIME_ZONE = DateTimeZone.forID("America/Bahia_Banderas");
 
-    public static DistributedQueryRunner createQueryRunner(TpchTable<?>... tables)
+    public static DistributedQueryRunner create()
             throws Exception
     {
-        return createQueryRunner(ImmutableList.copyOf(tables));
+        return builder().build();
     }
 
-    public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> tables)
-            throws Exception
+    public static Builder builder()
     {
-        return createQueryRunner(tables, ImmutableMap.of(), Optional.empty());
+        return new Builder();
     }
 
-    public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> tables, Map<String, String> extraProperties, Optional<Path> baseDataDir)
-            throws Exception
+    public static class Builder
+            extends DistributedQueryRunner.Builder
     {
-        return createQueryRunner(tables, extraProperties, ImmutableMap.of(), baseDataDir);
-    }
+        private Map<String, String> extraHiveProperties = ImmutableMap.of();
+        private List<TpchTable<?>> tables = ImmutableList.of();
 
-    public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> tables, Map<String, String> extraProperties, Map<String, String> extraHiveProperties, Optional<Path> baseDataDir)
-            throws Exception
-    {
-        assertEquals(DateTimeZone.getDefault(), TIME_ZONE, "Timezone not configured correctly. Add -Duser.timezone=America/Bahia_Banderas to your JVM arguments");
-        setupLogging();
+        protected Builder()
+        {
+            super(createSession(Optional.of(new SelectedRole(ROLE, Optional.of("admin")))));
+        }
 
-        DistributedQueryRunner queryRunner = DistributedQueryRunner
-                .builder(createSession(Optional.of(new SelectedRole(ROLE, Optional.of("admin")))))
-                .setNodeCount(4)
-                .setExtraProperties(extraProperties)
-                .setBaseDataDir(baseDataDir)
-                .build();
+        public Builder setExtraHiveProperties(Map<String, String> extraHiveProperties)
```

Comment:
`setHiveProperties`? not sure why we had "extra" here

Reply:
Because there are some defaults. And it was named like this before.